pywbem/cim_obj.py

```diff
@@ -2733,28 +2759,40 @@ def update(self, *args, **kwargs):
 
     def update_existing(self, *args, **kwargs):
         """
-        Update already existing properties of this CIM instance.
+        Update the values of already existing properties of this CIM instance.
 
-        Existing properties will be updated, and new properties will be
-        ignored without further notice.
+        From the specified new properties, only those properties will be
+        updated that already exist in the instance. No new properties will
+        be added to the instance, ignoring their specified new properties
```

Comment:
Change "ignoring their specified new properties without further notice" to something like ; "they will be ignored without further notice".

Change "only those properties will be only those properties will be updated that already exist in the instance to"
TO
"only properties that already exist in in the instance will be updated"

Reply:
DONE - changed the sentences to:
"From the specified new properties, only properties that already exist in the instance will be updated. New properties will not be added to the instance; they will be ignored without further notice."